### ISSUE TITLE
atft library doesn't export its members in TypeScript

### ISSUE THREAD

**Issue by ComFreek** [makimenko/angular-template-for-threejs#39]
**Describe the bug**
https://github.com/makimenko/angular-template-for-threejs/blob/master/projects/atft/src/public_api.ts doesn't export all the directives and components.

Implications are

1. you cannot use those directives in the app, e.g. if you want a `@ViewChild(WebGLRenderingComponent)` (as in my test code for #36), and
2. in `app.component.html` you'll see lots of errors shown by Visual Studio Code:
   ![image](https://user-images.githubusercontent.com/1827709/56298398-5db65400-6132-11e9-96de-7017f8d99dbb.png)

**Expected behavior**
Export all directives and components in `public_api.ts`. Now that we have a public package published, we should be cautious with modify the API without adhering to semver. If in doubt, make all properties/methods private first.

**Suggested fix**
For every subdirectory of `projects/atft/lib` create an `index.ts` exposing the public API. E.g. `projects/atft/lib/cameras/index.ts`, then in `public_api.ts` do `export * from './lib/cameras`.

**Comment by makimenko** [comment]
Please check, 
I hope it's ok now

**Comment by makimenko** [comment]
@ComFreek added you as collaborator to npm library (invitation sent to email). Feel free to publish changes to npm.

**Comment by ComFreek** [comment]
@makimenko Looks good to me! Thanks for your trust in me. I didn't get any email though! I've already checked all my other email addresses and junk folders :smile:

To adhere to semver, we should publish the changes as v0.1.0. Also the tag v0.0.3 for atft@0.0.2 is confusing, see the newly created `CHANGELOG.md`. I've drafted a GitHub release: https://github.com/makimenko/angular-template-for-threejs/releases Does it look okay for you?
